### ISSUE TITLE
Alternative for winston logger

### ISSUE THREAD

**Issue by andrew-aladev** [thaliproject/Thali_CordovaPlugin#1206]
We are using custom transport for `winston`. It can will receive `message` and `meta` arguments.

```
logger.debug('fit'); // -> message = 'fit'; meta = {};
logger.debug(new Error('fit')); // message = ''; meta = new Error('fit');
logger.debug(new Error('fit'), { a : 1 }); // message = new Error('fit'); meta = { a : 1 };
```

This looks looks like a bug, but this is a feature. [`exports.log` function](https://github.com/winstonjs/winston/blob/master/lib/winston/common.js#L136) is full of such "features".

`winston` looks unreliable, we want to find a good alternative for it.

**Comment by yaronyg** [comment]
Now would not be the best time to switch loggers if we can avoid it. Can we patch this to detect when we are getting an Error object and toString it?

**Comment by andrew-aladev** [comment]
Yes, we've patched and merged it. This is a part of [branch](https://github.com/thaliproject/Thali_CordovaPlugin/pull/1204).
